e2e/nx-misc/src/workspace.test.ts

```diff
@@ -759,7 +759,7 @@ describe('Workspace Tests', () => {
 
       expect(error).toBeDefined();
       expect(error.stdout.toString()).toContain(
-        `${lib1} is still depended on by the following projects`
```

Comment:
This is failing due to changes here https://github.com/nrwl/nx/commit/70d3728457f6ecf1c764ffef14e4a317e4d941bb